### PR TITLE
[RHPAM-3640] Configure Kafka JBPM Emitter from Operator

### PR DESCRIPTION
**Pull request by desmax74**
See: https://issues.redhat.com/browse/RHPAM-3640
Signed-off-by: desmax74 <mdessi@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by desmax74** [comment]
@spolti @jakubschwan

File: kieserver/modules/kieserver/module.yaml [hunk 1]
```diff
@@ -36,6 +36,9 @@ artifacts:
 - name: "slf4j-simple.jar"
   # slf4j-simple-1.7.22.redhat-2.jar
   md5: "62cc6eeb72e2738e3acc8957ca95f37b"
+- name: "jbpm-event-emitters-kafka-7.x.Final.jar"
+  # jbpm-event-emitters-kafka-7.52.0.Final-redhat-00004.jar
```
Comment (jakubschwan) [inline kieserver/modules/kieserver/module.yaml:40]: wouldn't be better to use there the name with version? @spolti 
Also aren't there needed some extra changes for future (auto) updates - new versions?

Comment (spolti) [reply kieserver/modules/kieserver/module.yaml:40]: @almope is this jar built on brew?
can we get it using brew calls?

File: kieserver/modules/kieserver/install [hunk 2]
```diff
@@ -25,6 +25,8 @@ if [[ -f "${SOURCES_DIR}/${BUSINESS_CENTRAL_DISTRIBUTION_ZIP}" ]] && [[ "${JBPM_
     fi
 fi
 
+mv /tmp/artifacts/jbpm-event-emitters-kafka-7.x.Final.jar "/opt"
```
Comment (spolti) [inline kieserver/modules/kieserver/install:28]: please use a more specific dir as I mentioned you on gchat.

Comment (spolti) [reply kieserver/modules/kieserver/install:28]: did you test cekit _dest_   parameters instead? https://docs.cekit.io/en/latest/descriptor/image.html#artifact-types

**Review by spolti — CHANGES_REQUESTED** [review]

File: kieserver/modules/kieserver/module.yaml (hunk 1 above)
Comment (spolti) [reply kieserver/modules/kieserver/module.yaml:40]: @desmax74 remember to update the cacher as well to update this jar when pushing automated PRs.
https://github.com/kiegroup/kie-cloud-tools/tree/master/cekit-helpers

File: kieserver/modules/kieserver/install (hunk 2 above)
Comment (desmax74) [reply kieserver/modules/kieserver/install]: Applied

File: kieserver/modules/kieserver/module.yaml (hunk 1 above)
Comment (desmax74) [reply kieserver/modules/kieserver/module.yaml:40]: ok

**Comment by desmax74** [comment]
Replaced by:https://github.com/jboss-container-images/rhpam-7-image/pull/488

**Pull request closed without merging**